### ISSUE TITLE
Can't upload file if the content is too large

### ISSUE THREAD

**Issue by Luna-Klatzer** [SYP-AHIF-2023-24-25/SolarDoc#202]
Currently, when attempting to upload a file to the server PostgreSQL returns an error indicating the content is too large (>255). This is an anomaly though because when editing a file normally this error does not happen and any length of file is accepted by the database.

Error:

```
ERROR 22001 (string_data_right_truncation) value too long for type character varying(255)
```

**Comment by Luna-Klatzer** [comment]
We thank Phoenix for literally saving us. Praise the Phoenix and Elixir gods for their high-performance caches. 💜

**Comment by Luna-Klatzer** [comment]
Fixed by #207.
